index.css

```diff
@@ -0,0 +1,147 @@
+body {
+  overflow: hidden;
+  background: #3b0e68;
+  background: radial-gradient(circle, #3b0e68 0%, #1a084e 100%);
+}
+
+.center-text {
+  text-align: center;
+  color: white;
+}
+
+.bottom-text {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 100px;
+  color: white;
+  text-align: center;
+}
+
+.mic {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+  cursor: pointer;
+  /* Adding cursor style to indicate it's clickable */
```

Comment:
Consider consolidating common styles into reusable classes for better maintainability [important]

- Generated by [LGTM](https://lgtm.pro)